src/Database/DatabaseConfig.php

```diff
@@ -100,6 +101,11 @@ public function getTemplateConnectionName(): string
             ?? config('tenancy.database.central_connection');
     }
 
+    public function getTenantHostConnectionName(): ?string
+    {
+        return config('tenancy.database.tenant_host_connection_name');
+    }
```

Comment:
When would this be null? And what's the behavior if it's null?

Reply:
> When would this be null? 

This can be null if developer does provide value in config. 

> And what's the behavior if it's null?

We are not handling null behavior yet. If config returns null, we should provide a default value. 

I think I got your point. This can't be null. The config key must have value. Leaving the key null is not a good option here, IMO. So I'm removing the nullable option from here.